src/main/java/codeu/model/data/User.java

```diff
@@ -32,11 +34,16 @@
    * @param passwordHash the password hash of this User
    * @param creation the creation time of this User
```

Comment:
Lets add the documentation for the "description" parameter

Reply:
Fixed it.